ibc_data/contrasts/Wedge.csv

```diff
@@ -0,0 +1,9 @@
+condition,lower_meridian-fixation,upper_meridian-fixation,left_meridian-fixation,right_meridian-fixation,lower_left-fixation,upper_left-fixation,lower_right-fixation,upper_right-fixation

```

Comment:
I could not find a longer description for this contrasts, and in the documentation each one is described as *vs. fixation* but I am nos sure we're actually modeling these fixations periods, since it's not an explicit condition. So maybe adding a line with `fixation` and -1 for each contrast?

Same applies to Ring task.

Reply:
No if there is no fixation condition modeled, we should not add it. The naming is however somewhat inconsistent...

Reply:
I modified the contrast names to match those in the script.
I hope this is the matrix we want? Just with 1 on every contrast column.